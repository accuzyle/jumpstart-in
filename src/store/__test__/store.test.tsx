import { store } from '..'

describe('store', () => {
  it('should match initial state', () => {
    expect(store.getState()).toMatchSnapshot()
  })
})
