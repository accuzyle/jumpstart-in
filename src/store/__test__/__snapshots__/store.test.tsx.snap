// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store should match initial state 1`] = `
Object {
  "collection": Array [],
  "composition": Object {
    "decks": Array [],
    "pool": Array [],
  },
  "settings": Object {
    "includedSets": Array [
      "ONE",
      "J22",
      "BRO",
      "DMU",
      "JMP",
    ],
    "paletteMode": "dark",
    "storeVersion": "v1",
  },
}
`;
